src/index.js

```diff
@@ -126,6 +126,14 @@ async function login(fields, fingerprint) {
     await this.notifySuccessfulLogin()
     return true
   }
+  if (
+    fingerPrintResp.body
```

Comment:
You can make a variable with     fingerPrintResp.body.html() only once and use it with includes then match